testing/test_end_to_end.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import subprocess
+import glob
+
+
+class TestEndToEnd(unittest.TestCase):
+
+    # Unit test for validating test outputs from install to test script
+    def test_endtoend(self):
+        generated_fa_files = []
+
+        # Fetch the generated fa files and verify that they match reference files
+        for filename in glob.glob("../test_output/*.fa"):
+            if filename.__contains__("msa") or filename.__contains__("refseq"):
+                generated_fa_files.append(open(filename, 'r'))
+
+        reference_fa_files = [open("test_constants/rpoB_0_msa_0.fa", 'r'),
+                              open("test_constants/rpoB_0_refseq_msa_1.fa", 'r')]
+
+        # compare the two FA files with reference
+        if len(generated_fa_files) > 0:
+
+            for i in range(0, 2):
```

Comment:
@KyleBehiels A better way to do this would be to take a md5 hash of each file and compare them that way.

https://stackoverflow.com/questions/36873485/compare-md5-hashes-of-two-files-in-python
https://docs.python.org/2/library/hashlib.html#module-hashlib
https://en.wikipedia.org/wiki/Md5sum